pac4j-oauth/src/test/java/org/pac4j/oauth/run/RunLinkedIn2Client.java

```diff
@@ -38,9 +36,9 @@ protected String getPassword() {
     @Override
     protected IndirectClient getClient() {
         final LinkedIn2Client client = new LinkedIn2Client();
-        client.setKey("gsqj8dn56ayn");
-        client.setSecret("kUFAZ2oYvwMQ6HFl");
-        client.setScope("r_basicprofile r_emailaddress rw_company_admin w_share");
+        client.setKey("86xtdkj897xk7d");
```

Comment:
It's the key and secret of a demo app, right?

Reply:
es these are the keys and the secret of the test I created with my
credentials. I would strongly suggest to
  a) create another app that is v2 capable with your credentials - or -
  b) update your current pack4j test app in order to be v2 capable.

Reply:
No problem. We'll do that when the need arises. So far, I just wanted to be sure it was no private key/secret.